nl_server/loader.py

```diff
@@ -23,12 +26,30 @@
 import nl_server.embeddings_map as emb_map
 from nl_server.nl_attribute_model import NLAttributeModel
 from shared.lib.custom_dc_util import get_custom_dc_user_data_path
+from shared.lib.custom_dc_util import is_custom_dc
 from shared.lib.gcs import download_gcs_file
 from shared.lib.gcs import is_gcs_path
 from shared.lib.gcs import join_gcs_path
 
 _EMBEDDINGS_YAML = 'embeddings.yaml'
 _CUSTOM_EMBEDDINGS_YAML_PATH = 'datacommons/nl/custom_embeddings.yaml'
+_EMBEDDINGS_SPEC_PATH: str = '/datacommons/nl/embeddings_spec.json'
+
+
+@dataclass
+class EmbeddingsSpec:
+  default_index: str
+  enabled_indexes: List[str]
+  vertex_ai_model_info: Dict[str, any]
+  ranking_enabled: bool
```

Comment:
nit: should this be in config.py along with other configs?

Reply:
i kept this here because it's only used in the loader, can move if you prefer

Reply:
ok, sg